### ISSUE TITLE
Github action doesn't create a release

### ISSUE THREAD

**Issue by jackjansen** [cwi-dis/vr2gather-orchestrator-v2#35]
When I add a tag the docker image is built and pushed, but no release is created.

**Comment by troeggla** [comment]
Yeah, you have to do that manually. This is because you have to write the release notes either way.
